pkgs/development/python-modules/pypiserver/default.nix

```
@@ -0,0 +1,40 @@
+{ buildPythonPackage, fetchFromGitHub, lib, passlib, pytestCheckHook, setuptools
+, setuptools-git, twine, webtest }:
+
+buildPythonPackage rec {
+  pname = "pypiserver";
+  version = "1.4.2";
+
+  src = fetchFromGitHub {
+    owner = pname;
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1z5rsmqgin98m6ihy1ww42fxxr6jb4hzldn8vlc9ssv7sawdz8vz";
+  };
+
+  natibeBuildInputs = [ setuptools-git ];
```

```suggestion
  nativeBuildInputs = [ setuptools-git ];
```

Typo, did you test this?

---

Interestingly I did... I'll take another look and see if maybe it's not actually needed.

---

It is needed!

---

Hah, yep, confirmed! Will fix up. Sorry for the mess, have been trying to patch this up in between doing a bunch of other things.